warning: [removal] Long(long) in Long has been deprecated and marked for removal

Folgende Warnung tritt beim bauen auf und sollte mal beurteilt werden:
```
> Task :compileJava
/builds/mv/MServer/src/main/java/de/mediathekview/mlib/tool/Datum.java:61: warning: [removal] Long(long) in Long has been deprecated and marked for removal
        final int ret = new Long((this.getTime() - new Datum().getTime()) / (1000)).intValue();
                        ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 warning
```

--- a/src/main/java/de/mediathekview/mlib/tool/Datum.java
+++ b/src/main/java/de/mediathekview/mlib/tool/Datum.java
@@ -1,9 +1,9 @@
-/*    
+/*
  *    MediathekView
  *    Copyright (C) 2008   W. Xaver
  *    W.Xaver[at]googlemail.com
  *    http://zdfmediathk.sourceforge.net/
- *    
+ *
  *    This program is free software: you can redistribute it and/or modify
  *    it under the terms of the GNU General Public License as published by
  *    the Free Software Foundation, either version 3 of the License, or
@@ -24,50 +24,22 @@ import java.util.Date;
 
 @SuppressWarnings("serial")
 public class Datum extends Date {
-    private final static SimpleDateFormat dateFormatter1 = new SimpleDateFormat("dd.MM.yyyy");
-    private final static SimpleDateFormat dateFormatter2 = new SimpleDateFormat("yyyy.MM.dd");
+  private static final SimpleDateFormat dateFormatter1 = new SimpleDateFormat("dd.MM.yyyy");
 
-    public Datum() {
-        super();
-    }
+  public Datum() {
+    super();
+  }
 
-    public Datum(long l) {
-        super(l);
-    }
+  public Datum(long l) {
+    super(l);
+  }
 
-    @Override
-    public String toString() {
-        if (this.getTime() == 0) {
-            return "";
-        } else {
-            return dateFormatter1.format(this);
-        }
+  @Override
+  public String toString() {
+    if (this.getTime() == 0) {
+      return "";
+    } else {
+      return dateFormatter1.format(this);
     }
-
-    public String toStringR() {
-        if (this.getTime() == 0) {
-            return dateFormatter2.format(new Date());
-        } else {
-            return dateFormatter2.format(this);
-        }
-    }
-
-    /**
-     * Liefert den Betrag der Zeitdifferenz zu jetzt.
-     *
-     * @return Differenz in Sekunden.
-     */
-    public int diffInSekunden() {
-        final int ret = new Long((this.getTime() - new Datum().getTime()) / (1000)).intValue();
-        return Math.abs(ret);
-    }
-
-    /**
-     * Liefert den BETRAG! der Zeitdifferenz zu jetzt.
-     *
-     * @return Differenz in Minuten.
-     */
-    public int diffInMinuten() {
-        return (diffInSekunden() / 60);
-    }
+  }
 }

--- a/src/main/java/de/mediathekview/mlib/tool/DatumFilm.java
+++ b/src/main/java/de/mediathekview/mlib/tool/DatumFilm.java
@@ -1,9 +1,9 @@
-/*    
+/*
  *    MediathekView
  *    Copyright (C) 2008   W. Xaver
  *    W.Xaver[at]googlemail.com
  *    http://zdfmediathk.sourceforge.net/
- *    
+ *
  *    This program is free software: you can redistribute it and/or modify
  *    it under the terms of the GNU General Public License as published by
  *    the Free Software Foundation, either version 3 of the License, or
@@ -19,56 +19,39 @@
  */
 package de.mediathekview.mlib.tool;
 
-import de.mediathekview.mlib.tool.Datum;
-
 import java.text.SimpleDateFormat;
-import java.util.Date;
 import java.util.TimeZone;
 
 @SuppressWarnings("serial")
 public class DatumFilm extends Datum {
-    // die Filme werden immer in der Zeitzone "Europe/Berlin" gesucht
+  // die Filme werden immer in der Zeitzone "Europe/Berlin" gesucht
 
-    private final static SDF dateFormatter1 = new SDF("dd.MM.yyyy");
-    private final static SDF dateFormatter2 = new SDF("yyyy.MM.dd");
+  private static final SDF dateFormatter1 = new SDF("dd.MM.yyyy");
 
-    public DatumFilm() {
-        super();
+  public DatumFilm(long l) {
+    super(l);
+  }
+
+  @Override
+  public String toString() {
+    if (this.getTime() == 0) {
+      return "";
+    } else {
+      return dateFormatter1.format(this);
+    }
+  }
+
+  private static class SDF extends SimpleDateFormat {
+    private static final TimeZone tz = TimeZone.getTimeZone("Europe/Berlin");
+
+    public SDF() {
+      super();
+      this.setTimeZone(tz);
     }
 
-    public DatumFilm(long l) {
-        super(l);
+    public SDF(String str) {
+      super(str);
+      this.setTimeZone(tz);
     }
-
-    @Override
-    public String toString() {
-        if (this.getTime() == 0) {
-            return "";
-        } else {
-            return dateFormatter1.format(this);
-        }
-    }
-
-    @Override
-    public String toStringR() {
-        if (this.getTime() == 0) {
-            return dateFormatter2.format(new Date());
-        } else {
-            return dateFormatter2.format(this);
-        }
-    }
-
-    private static class SDF extends SimpleDateFormat {
-        private final static TimeZone tz = TimeZone.getTimeZone("Europe/Berlin");
-
-        public SDF() {
-            super();
-            this.setTimeZone(tz);
-        }
-
-        public SDF(String str) {
-            super(str);
-            this.setTimeZone(tz);
-        }
-    }
+  }
 }

